### ISSUE TITLE
ANDROID-79920582 - *.gradle.kts files don't show up in Android Studio view of project

### ISSUE THREAD

**Issue by erikhuizinga** [gradle/kotlin-dsl#884]
### Expected Behavior

I expect to see Kotlin DSL Gradle scripts in the Android view of an Android project in Android Studio.

### Current Behavior

Currently, *.gradle.kts files aren't shown under the Gradle Scripts section in Android Studio with the Android project view.

### Context

The following screenshots are of the same project; the first is the Android view, the second is the Project view. Note that in the second a file called settings.gradle.kts is visible where in the first it is not, while I expected it to be. I haven't tested this in other IDEs or with other views.

| Android view | Project view (some parts cropped for privacy) |
|--------------|--------------|
| <img width="206" alt="schermafbeelding 2018-05-17 om 11 27 49" src="https://user-images.githubusercontent.com/19374736/40169173-b213d98e-59c5-11e8-88ac-304c8a23da49.png">            | <img width="200" alt="schermafbeelding 2018-05-17 om 11 42 51" src="https://user-images.githubusercontent.com/19374736/40170001-b5518bc6-59c7-11e8-89d6-0b32d6f469c7.png">            |

### Your Environment

 * Build scan URL: ???
 * Gradle distro: v4.7

Others:

```
Android Studio 3.1.2
Build #AI-173.4720617, built on April 13, 2018
JRE: 1.8.0_152-release-1024-b01 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.13.4
```

#### Vendor issue

> Looks like it's been fixed in Android Studio 3.3 Beta 4.

https://issuetracker.google.com/issues/79920582

**Comment by StefMa** [comment]
That is nothing the project can do (I think).
The Android View is created and managed by Google.
I think you have to open a issue at there issue tracker...

**Comment by JLLeitschuh** [comment]
Please do link to an issue with Jetbrains or Google when one is opened there.

**Comment by erikhuizinga** [comment]
https://issuetracker.google.com/issues/79920582

**Comment by eskatos** [comment]
Fixed in Studio 3.3, closing now that it is stable
